packages/amplify_authenticator/example/integration_test/sign_in_with_email_test.dart

```diff
@@ -79,49 +74,50 @@ void main() {
       await signInPage.expectUserNotFound();
     });
 
+    // TODO(dnnoyes): enable subscriptions with graphql package
```

Comment:
Can we just disable these on non-mobile platforms? You can use the `skip` flag on the group/test.

Reply:
@dnys1 We can - but I believe we would conditionally need to add the AmplifyApi plugin for those platforms as well. If you and Jordan thinks that's worth it, I can make the change.

Reply:
Is it just a matter of conditionally setting `additionalConfigs`?
```dart
additionalConfigs: !kIsWeb && (Platform.isIOS || Platform.isAndroid) ? [AmplifyAPI()] : null,
```